src/codeflare_sdk/ray/cluster/cluster.py

```diff
@@ -744,6 +802,20 @@ def _create_resources(yamls, namespace: str, api_instance: client.CustomObjectsA
     )
 
 
+def _apply_resources(
+    yamls, namespace: str, api_instance: client.CustomObjectsApi, force=False
+):
+    api_instance.server_side_apply(
+        field_manager="cluster-manager",
```

Comment:
This should probably be `cf-sdk` or something similar and pulled out into a variable, used both here and for the AppWrapper